packages/appkit/src/client.ts

```diff
@@ -545,7 +537,7 @@ export class AppKit {
     return this.initPromise
   }
 
-  private extendCaipNetworksWithImages(
+  private prepareCaipNetworks(
```

Comment:
renaming this because it is not only adding images but also extending the rpc url with `projectId`